P_huljek

DrHrast opened the pull request:
Malo fali ali samo komentara u kojima sam se pogubio. Tajmer ne radi i imao sam nekih konfliktnih comitova pa sam revertao na stari i zalijepio main.cpp, a ostalo ne, ali nije bilo nista pametno tamo.

DrHrast committed:
P_huljek

Vježba 4.

DrHrast committed:
Some error

DrHrast committed:
Implemented: projectile movement, projectile colision, score count, high score, game over mechanics on ship with obstacle colision...

ncaklovic commented:
Meni je vaš timer proradio otprve (otkomentirao sam sav zakomentirani kod) kad sam dodao ove dvije linije: [nwpwin.h](https://github.com/vsite-nwp/src/blob/main/p4/nwpwin.h#L48), [nwpwin.cpp](https://github.com/vsite-nwp/src/blob/main/p4/nwpwin.cpp#L82). Naravno, mora biti i `main_window::on_timer(int id)`.

DrHrast committed:
Timer implemented.
/* Igrajte pa vidite. Moj high score je 41 u 90 sekundi pa samo da znamo što će biti vama. */
